packages/lit-dev-content/site/tutorials/content/carousel/05.md

```diff
@@ -0,0 +1,57 @@
+It works! But it's a little boring, and changing the selected item is a bit
+jarring. This is a great place to use animation to provide feedback about what's
+happening. Animating elements can be tricky, so we'll tackle it in a few steps.
+First, we need to modify the DOM and styling a bit to prepare for adding the
+animation.
+
+We want to slide the selected item into view from the left or right based on
+whether or not the next item is greater than the current one. To do this,
+we'll add a slot for the `previous` element. We'll use this to manage the
+positions of the previously and currently selected items. Then we'll add a
+"container" element surrounding the items that we'll animate left and right.
+We'll move our click handler there to make sure we don't miss any clicks.
+
+We're going to position `left` property of the "container" and the `previous`
+and `selected` slots. To setup for that, we've already imported
+the `styleMap` directive and we'll use it in the template. For now just
+stub out a private `left` property that we'll use to track position and
+different values we'll map into the styles of those elements: `animateLeft`,
+`selectedLeft`, and `previousLeft`. We'll fix those up in the next step.
```

Comment:
I think this paragraph suffers greatly from "instruction overload". It would be beneficial to break these instructions up with code snippets.

Additionally there is some jargon, specifically:

* ``going to position `left` property``
* ``stub out a private `left` property``

Reply:
Reworded

Reply:
Reworded.